roadside-forms-frontend/frontend_web_app/src/components/Event/createEvent.js

```diff
@@ -373,6 +390,18 @@ export const CreateEvent = () => {
         );
       }
     } else {
+      if (eventCreationFailed) {
```

Comment:
will this only show up on the police copy?

Reply:
eventCreationFailed can only be set to true by the onSubmit function in the API call, so yes.